scss/_patterns_footer.scss

```diff
@@ -22,10 +38,6 @@
 
     &__nav {
       margin-top: 0;
-
-      p + & {
-        margin-top: -$sp-unit * 2;
```

Comment:
This was meant to keep paragraphs closer together, as mostly these are one liners that look too far apart. Was it breaking something?